### PR TITLE
Feat/light 4j/#391 support caching multiple Jwts

### PR DESCRIPTION
**Pull request by BalloonWen**
related draft RFC https://github.com/networknt/light-rfcs/blob/237abdd13b4faa5e2e834054a65002b865cd959b/light-4j/0016-Router-Client-Accepting-Provided-Scopes-for-Auth.md

Provided a token manager to issue JWT and cache JWT globally.
Provided an ability to send different requests based on different Oauth server.

### CODE REVIEW

File: client/src/main/resources/config/client.yml [hunk 1]
```diff
@@ -16,9 +16,9 @@ tls:
   # key store location
   keyStore: client.keystore
 # settings for OAuth2 server communication
-cache:
-  capacity: 200
 oauth:
+  cache:
+    capacity: 200
```
[the remark this answers is not in the document]
Comment (BalloonWen) [reply client/src/main/resources/config/client.yml]: moved under token config.

**Comment by NicholasAzar** [comment]
@BalloonWen could you please fix the conflicts?

**Comment by BalloonWen** [comment]
@NicholasAzar merged.

**Comment by NicholasAzar** [comment]
@BalloonWen it's still showing one file in conflict.. also why the build is failing on this?

**Comment by BalloonWen** [comment]
@NicholasAzar The conflict was because Steve changed some imports on that file too a few hours ago.
The build was successful on my local before I push. This time I build it again and it also successes.
This test case Http2ClientTest.testMixed has chances to fail or pass on the Travis CI build for both me and @jiachen1120  But it always passes on our locals.

**Comment by BalloonWen** [comment]
@NicholasAzar if I changed multithread from 4 to 50, it will fail because of ClosedChannelException. It works fine with 10 threads. I'll find more details.

**Comment by stevehu** [comment]
I tried to build on my local and it worked fine. Also triggered the travis-ci and it passed.

**Comment by NicholasAzar** [comment]
[support caching multiple Jwts](https://trello.com/c/WHpHXJID/127-support-caching-multiple-jwts)

**Pull request merged**